modules/navier_stokes/test/tests/finite_volume/ins/block_restriction/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  issues = '#16972'
+  [block_restricted_variables]
+    type = 'Exodiff'
+    input = 2d-rc.i
+    exodiff = 2d-rc_out.e
+    requirement = 'The system shall be able to restrict block-restrict all variables in a heated channel simulation with passive scalar advection.'
```

Comment:
Weird I can't make a suggestion...maybe because I'm looking at this commit by commit. Anyway probably want to remove one of those "restricts".

Thank you very much for adding this test!